specifyweb/frontend/js_src/lib/components/TreeView/Row.tsx

```diff
@@ -63,7 +69,12 @@ export function TreeRow({
   );
 
   React.useEffect(() => {
-    if (Array.isArray(focusPath) && focusPath.length === 0) setFocusedRow(row);
+    if (
+      Array.isArray(focusPath) &&
+      focusPath.length === 0 &&
+      lastTreeClicked === 'first'
```

Comment:
this doesn't look right
1. only the first tree can set focused row? 2nd tree can't? that doesn't seem right
2. even if that was desired, instead of checking for that here, the check should be inside the setFocusedRow function, not here
3. but really, both trees should be able to have independent focused row states, and calling setFocusedRow updates correct state - no checks should be needed here

Reply:
ideally, the tree shouldn't even know if it's first or 2nd or if it's focused or not - the difference in behavior would be facilitated by providing different callbacks, not by providing magic flags like "isFirst"